main.py

```diff
@@ -343,6 +357,28 @@ def closeEvent(self, event):
         self.can_bus.disconnect_bus()
         event.accept()
 
+    def create_ids(self):
```

Comment:
This function should probably go in either a faults.py, utils.py, or get grouped in with the daq_protocol.py  stuff. Either way, I’d rather it I not be in main.py.

Reply:
Also there are merge conflicts, you may have to merge master in to resolve